tests/test_api.py

```diff
@@ -0,0 +1,119 @@
+import os           # used to get the location of the testdata
+import pygeoj       # used to parse the geojson file
+import shapefile 
+import xarray as xr
+import sys
+
+__location__ = os.path.realpath(os.path.join(os.getcwd(), os.path.dirname(__file__)))
+
+###############################
+# Check file existence
+###############################
+def test_geojson_file_exist():
+    filepath=__location__+"/Testdata/muenster_ring_zeit.geojson"
+    assert os.path.isfile(filepath) == True
+
+def test_nc_file_exist():
+    filepath=__location__+"/Testdata/ECMWF_ERA-40_subset1.nc"
+    assert os.path.isfile(filepath) == True
+
+def test_kml_file_exist():
+    filepath=__location__+"/Testdata/aasee.kml"
+    assert os.path.isfile(filepath) == True
+
+def test_tif_file_exist():
+    filepath=__location__+"/Testdata/wf_100m_klas.tif"
+    assert os.path.isfile(filepath) == True
+
+def test_gpkg_file_exist():
+    filepath=__location__+"/Testdata/nc.gpkg"
+    assert os.path.isfile(filepath) == True
+
+def test_csv_file_exist():
+    filepath=__location__+"/Testdata/cities_NL.csv"
+    assert os.path.isfile(filepath) == True
+
+def test_gml_file_exist():
+    filepath=__location__+"/Testdata/clc_1000_PT.gml"
+    assert os.path.isfile(filepath) == True
+
+def test_shp_file_exist():
+    filepath=__location__+"/Testdata/Abgrabungen_Kreis_Kleve_Shape.shp"
+    assert os.path.isfile(filepath) == True
+
+def test_dbf_file_exist():
+    filepath=__location__+"/Testdata/Abgrabungen_Kreis_Kleve_Shape.dbf"
+    assert os.path.isfile(filepath) == True
+
+###############################
+# Check bbox extraction
+###############################
+def test_geojson_extract_bbox():
+    filepath=__location__+"/Testdata/muenster_ring_zeit.geojson"
+    geojson = pygeoj.load(filepath)
```

Comment:
The library name should be `geoextent` not `pygeoj`.

Reply:
Done.